README.md

```diff
@@ -176,6 +176,12 @@ Normally an [`anchor element`](https://developer.mozilla.org/en-US/docs/Web/HTML
 
 There are many advantages of using an anchor element, the main one being accessibility.
 
+Alternatively, if you would still like to allow relative links to other parts of your site to navigate as normally, you can opt out of this behavior on a link-by-link basis:
```

Comment:
I use a readme generator which means the `README.md` file is actually based on the `blueprint.md` file. Would you be able to add the documentation to the `blueprint.md` file instead and run `npm run readme` to generate a new `README.md` file?  I know this hasn't been documented anywhere - I will make sure to do it for future PRs! 👍

Reply:
done! https://github.com/andreasbm/router-slot/pull/5/commits/d713a3053729607be67744fea2ac5e54ba0bfcba